lib/flexirest/request.rb

```diff
@@ -224,7 +224,7 @@ def call(explicit_parameters=nil)
 
     def prepare_params
       params = @params || @object._attributes rescue {}
-      if params.is_a?(String) || params.is_a?(Fixnum)
+      if params.is_a?(String) || params.is_a?(Integer)
```

Comment:
Does this now only work for Ruby 2.4, or is this change compatible with 1.9.3 and upwards as per the rest of Flexirest?